projects/aca-content/src/lib/services/node-actions.service.ts

```diff
@@ -64,7 +65,9 @@ export class NodeActionsService {
     private apiService: AlfrescoApiService,
     private translation: TranslationService,
     private thumbnailService: ThumbnailService
-  ) {}
+  ) {
+    this.isNodeLockedSubject = new BehaviorSubject<boolean>(false);
```

Comment:
No need to assign it in constructor. You can do it inline next to declaration: 
```ts
private isNodeLockedSubject = new BehaviorSubject(false);

Reply:
done!